kyuubi-server/src/main/scala/org/apache/kyuubi/engine/hive/HiveProcessBuilder.scala

```diff
@@ -75,20 +81,21 @@ class HiveProcessBuilder(
     env.get("YARN_CONF_DIR").foreach(classpathEntries.add)
     // jars from hive distribution
     classpathEntries.add(s"$hiveHome${File.separator}lib${File.separator}*")
-    val hadoopCp = env.get("HIVE_HADOOP_CLASSPATH").orElse(env.get("HADOOP_CLASSPATH"))
-    hadoopCp.foreach(path => classpathEntries.add(s"$path${File.separator}*"))
-    if (hadoopCp.isEmpty) {
-      warn(s"HIVE_HADOOP_CLASSPATH or HADOOP_CLASSPATH don't export.")
+    val extraCp = conf.get(ENGINE_HIVE_EXTRA_CLASSPATH)
```

Comment:
Now, HiveSQLEngine is coupled of `kyuubi.engine.hive.extra.classpath` in production environment. If we don't set `kyuubi.engine.hive.extra.classpath` the HiveSQLEngine will error.